### PR TITLE
search modal fix, filtration is working

### PR DESCRIPTION
**Pull request by sabrina34352**
image

### CODE REVIEW

File: packages/frontapp/src/app/components/SearchModal/index.tsx [hunk 1]
```diff
@@ -82,7 +87,16 @@ const SearchModal: FC<IProps> = ({
   checkBoxHandler,
 }) => {
   const numberOfInitialItems = 7; // number of items to show before clicked ShowMore
-  const [showMore, setShowMore] = useState(false);
+  const [showMore, setShowMore] = useState<showMoreStats>({
+    Authors: false,
```
Comment (IvanUglovets) [inline packages/frontapp/src/app/components/SearchModal/index.tsx:91]: please use camelCase

File: jest.preset.js [hunk 2]
```diff
@@ -1,3 +1,8 @@
 const nxPreset = require('@nrwl/jest/preset');
 
-module.exports = { ...nxPreset };
+module.exports = {
+  ...nxPreset,
+  transformIgnorePatterns: [
+    'node_modules/(?!(react-native|my-project|redux-persist)/)',
```
Comment (unrealsolver) [inline jest.preset.js:6]: What is the purpose of this?

File: packages/frontapp/src/app/components/BooksByCategory/index.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import BookList from './bookList';
+import { screen } from '@testing-library/react';
+import { render } from '../../../helpers/customRender';
+import { mockMaterial } from '../../../helpers/mockData';
```
Comment (unrealsolver) [inline packages/frontapp/src/app/components/BooksByCategory/index.spec.tsx:4]: I'd rather kept it relative to this file.

**Review by IvanUglovets — APPROVED** [review]

**Review by unrealsolver — APPROVED** [review]

File: jest.preset.js (hunk 2 above)
Comment (sabrina34352) [reply jest.preset.js:6]: when testing redux-persist throws an error and disturbs testing, this is the only solution i found

**Pull request merged**